storagenode/storagenodedb/bandwidthdb.go

```diff
@@ -31,10 +33,49 @@ func (db *bandwidthdb) Add(ctx context.Context, satelliteID storj.NodeID, action
 		INSERT INTO
 			bandwidth_usage(satellite_id, action, amount, created_at)
 		VALUES(?, ?, ?, ?)`, satelliteID, action, amount, created)
-
+	if err == nil {
+		db.bandwidth.mu.Lock()
+		defer db.bandwidth.mu.Unlock()
+
+		beginningOfMonth := getBeginningOfMonth(created.UTC())
```

Comment:
So when it inserts a bandwidth entry for the previous month the whole cache rolls back?

Reply:
Updated.  Only bandwidth from the next month can reset the cache.

Removed cache updates from BandwidthUsed.  If it has the current months data in cache it returns it. if, queries for it.